cli/defenseclaw/commands/windows_native_uninstall.py

```
@@ -0,0 +1,760 @@
+    if not _INSTALL_STATE_REQUIRED_FIELDS.issubset(fields) or not fields.issubset(
+        _INSTALL_STATE_REQUIRED_FIELDS | _INSTALL_STATE_OPTIONAL_FIELDS
+    ):
+        raise NativeWindowsUninstallRefusal(
+            "Native installer state does not use the supported closed schema."
+        )
+    schema = state.get("schema_version")
+    version = state.get("version")
+    source_commit = state.get("source_commit")
+    transaction_id = state.get("transaction_id", "")
+    if schema != 1 or isinstance(schema, bool):
+        raise NativeWindowsUninstallRefusal("Native installer state schema is unsupported.")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)